code/modules/mob/living/simple_animal/hostile/thing.dm

```diff
@@ -0,0 +1,1381 @@
+		return
+	if(absorbing == 1)
+		return
+
+	if(creep_cooldown <= 0)
+		if(!isturf(loc) || istype(loc, /turf/space))
+			return
+		if(locate(/obj/structure/alien/creep/node) in get_turf(src))
+			return
+		visible_message("<span class='warning'>\the [src] has planted some creep!</span>")
+		new /obj/structure/alien/creep/node(loc)
+		creep_cooldown = initial(creep_cooldown)
+	else
+		to_chat(src, "<span class='warning'>This ability is on cooldown (Time remaining: [creep_cooldown]).</span>")
+
+/mob/living/simple_animal/hostile/thing/verb/Buildcreep()
+	set name = "Build - Create Building"
+	set category = "Unknown"
+	set desc = "Create a useful building on creep."
+
+	Spawncreepbuilding()
+
+/mob/living/simple_animal/hostile/thing/proc/Spawncreepbuilding() // Taken from alien_powers, resin()
+	if(stat == DEAD)
+		return
+
+	if(can_build == 0)
+		to_chat(src, "<span class='warning'>You cannot use this ability! Unlock it by spending 10 points, empowering yourself!</span>")
+		return
+
+	if(phasing == 1)
+		return
+	if (prying == 1)
+		return
+	if(absorbing == 1)
+		return
+
+	if(!locate(/obj/structure/alien/creep) in get_turf(src))
+		to_chat(src, "<span class='warning'>You need to be on creep to use this ability!</span>")
+		return
+
+	if(locate(/obj/structure/alien/thing) in get_turf(src))
+		to_chat(src, "<span class='warning'>There is a structure here already!</span>")
+		return
+
+	if(build_cooldown >= 1)
+		to_chat(src, "<span class='warning'>This ability is on cooldown (Time remaining: [build_cooldown]).</span>")
+		return
+
+	var/choice = input("Choose what you wish to create?","Creep building") as null|anything in list("gasser","passageway")
+
+	if(!choice)
+		to_chat(src, "<span class='warning'>Cancelled.</span>")
+		return
+
+	build_cooldown = initial(build_cooldown)
+	visible_message("<span class='warning'>\the [src] shapes the creep into a [choice]!</span>")
+	switch(choice)
+		if("gasser")
+			new /obj/structure/alien/thing/gasser(loc)
+		if("passageway")
+			new /obj/structure/alien/thing/spawner(loc)
+	return
+
+/mob/living/simple_animal/hostile/thing/verb/rally()
+	set name = "Rally Crawlers"
+	set category = "Unknown"
+	set desc = "Rally your crawlers towards you."
+
+	rally_crawlers()
+
+/mob/living/simple_animal/hostile/thing/proc/rally_crawlers()
+	if(stat == DEAD)
+		return
+
+	if(can_build == 0)
+		return
+
+	if(phasing == 1)
+		return
+
+	to_chat(src, "<span class='warning'>You rally your crawlers.</span>")
+	var/turf/T = get_turf(src)
+	var/list/surrounding_turfs = block(locate(T.x - 1, T.y - 1, T.z), locate(T.x + 1, T.y + 1, T.z))
+	if(!surrounding_turfs.len)
+		return
+
+	for(var/mob/living/simple_animal/hostile/creepcrawler/CR in GLOB.living_mob_list)
+		if(isturf(CR.loc) && get_dist(CR, T) <= 25)
+			CR.LoseTarget()
+			CR.Goto(pick(surrounding_turfs), CR.move_to_delay)
+	return
+
+/obj/machinery/door/airlock/attack_animal(mob/living/simple_animal/hostile/thing/user)  // Pry open doors with the armblade
```

Comment:
This must be an `UnarmedAttack` on your mob, we can't accept code that defines overrides on unrelated parts of the codebase.